flake-registry.json

```diff
@@ -298,116 +33,6 @@
         "repo": "nixpkgs",
         "type": "github"
       }
-    },
-    {
-      "from": {
-        "id": "templates",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "templates",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "patchelf",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "patchelf",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "poetry2nix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "poetry2nix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "nix-serve",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "edolstra",
-        "repo": "nix-serve",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "nickel",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "tweag",
-        "repo": "nickel",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "bundlers",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "bundlers",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "pridefetch",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "SpyHoodle",
-        "repo": "pridefetch",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "systems",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-systems",
-        "repo": "default",
-        "type": "github"
-      }
-    },
```

Comment:
Keep this one

Reply:
There is some utility to be able to write:
```
outputs = { self, nixpkgs, systems }: let eachSystem = nixpkgs.lib.genAttrs (import systems); in ...
```
Maybe it's not the cleanest, but it's nice to hack things quickly.

Making it easy is a good way to promote that pattern.

Reply:
As said already, the registry really shouldn't be used in a `flake.nix` neither by explicit `flke:` inputs, neither by just adding things to the argumentset of the output function that haven't been explicitely defined in the `inputs` attribute.

This uses the registry implicitely. In the eyes of many a misfeature of nix.

For the CLI though the `systems` entry in the registry is rather useless, as shown in my example. It became more usefull if the repository would be redesigned to use branches rather than several individual repositories.

It is not clear to me though, whom or how exactly to contact though, thats why I will blindly ping @nix-systems hoping thats a pingable user and not an org…